### ISSUE TITLE
Beginner-friendly docs

### ISSUE THREAD

**Issue by ryanraposo** [bendera/vscode-webview-elements#5]
First of all: this is so awesome! Thank you for making it. I'm far from new to extension development, but am essentially a beginner when it comes to web dev. I was about to just clone and tinker with it/figure out what I don't know, but I thought this might help you out...

I imagine a large demographic for the project will be people like me, who are looking for premade components _because_ of that lack of experience, though they might get spooked.

It might help adoption (and me!) if the docs are a little more approachable. :)

If you're like me, but the opposite, you might not know what's unapproachable about it. Let me know.

**Comment by bendera** [comment]
Thank you for your feedback! You're absolutely right, I should write better documentation. Creating and maintaining a component framework needs much more effort than I thought. Basically, I've created this project to provide UI components for my VSCode extension and I am the only one user. Until I finish the documentation this is a usage example:

**Comment by bendera** [comment]
(I closed the thread accidentally.)
https://github.com/bendera/vscode-commit-message-editor/tree/frontend-refactor
This is a feature branch that will be merged into the master. In this branch, I've rewritten the webview part to a component based frontend. I've created an emulated environment, so you can try the UI in the browser if you execute the npm run start command.
This is the advanced way but if you want to use it in static HTML just include the node_modules/@bendera.../dist/bundled.js into your page. And then you can use the custom elements on that page.

**Comment by bendera** [comment]
Hi,

I've created a "Getting Started" page, please share your thoughts. If you find a grammar mistake, or you would rephrase a sentence, don't hesitate to share with me. I'm not a native speaker.

https://bendera.github.io/vscode-webview-elements/pages/getting-started/

I am continuously working on other pages.

**Comment by ryanraposo** [comment]
> I've created a "Getting Started" page, please share your thoughts. If you find a grammar mistake, or you would rephrase a sentence, don't hesitate to share with me. I'm not a native speaker.

@bendera I would _love_ to help with that. :)

Talk soon.

**Comment by bendera** [comment]
I'm closing this ticket because of inactivity. Feel free to reopen anytime if you have any comments.
